src/components/results/shortcircuit/shortcircuit-analysis-result.type.ts

```diff
@@ -40,11 +38,46 @@ export interface SCAResultFault {
     feederResults: SCAResultFaultFeederResult[];
 }
 
-export interface ShortcircuitAnalysisResult {
-    resultUuid: UUID;
-    writeTimeStamp: Date;
-    faults: SCAResultFault[];
-}
+type Pageable = {
+    sort: {
+        sorted: boolean;
+        empty: boolean;
+        unsorted: boolean;
+    };
+    pageNumber: number;
+    pageSize: number;
+    offset: number;
+    paged: boolean;
+    unpaged: boolean;
+};
+
+export type ShortcircuitAnalysisResult = {
+    content: SCAResultFault[];
+    pageable: Pageable;
+    last: boolean;
+    totalPages: number;
+    totalElements: number;
+    first: boolean;
+    size: number;
+    number: number;
+    sort: {
+        sorted: boolean;
+        empty: boolean;
+        unsorted: boolean;
```

Comment:
why having `sorted` and `unsorted` at same time? they're always of opposed value no?

Reply:
it's coming from [Spring Data Pageable](https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/domain/Pageable.html), so nothing to do...